### ISSUE TITLE
Info Icon on the contact chat screen is missing TestId and AccessibilityID

### ISSUE THREAD

**Issue by nodlesh** [bcgov/bc-wallet-mobile#1029]
Go to the settings/Contacts and click a contact.
This icon is missing the TestID and Accessibility Label when in the Contacts messenger screen. 
![Screenshot 2023-04-05 at 12.21.52 PM.png](https://images.zenhubusercontent.com/5e3864b3b64bc2429a017886/0ff779d5-d966-4fc3-b916-ab7276641a0d)

Maybe while you are in there, make sure the user interactable elements on the contact info screen also has the appropriate TestIDs and Accessibility Labels.

**Comment by nodlesh** [comment]
Fixed on Android and iOS.
